exchanges/bitfinex/bitfinex.go

```diff
@@ -160,19 +160,33 @@ func (b *Bitfinex) GetV2FundingInfo(key string) (MarginV2FundingData, error) {
 	if err != nil {
 		return response, err
 	}
-	if len(resp) != 2 {
+	if len(resp) != 3 {
 		return response, errors.New("invalid data received")
 	}
-	avgRate, ok := resp[0].(float64)
+	sym, ok := resp[0].(string)
 	if !ok {
-		return response, errors.New("failed type assertion for rate")
+		return response, errors.New("failed type assertion for sym")
```

Comment:
whats sym?

Reply:
Dont know myself. Bitfinex labels it sym n thats about it. Everytime they just return "sym" n the json tag is "sym"